assets/dev/js/editor/controls/behaviors/scrubbing.js

```diff
@@ -0,0 +1,164 @@
+			scrubbingOverClass: 'e-scrubbing-over',
+			...userOptions,
+		};
+	}
+
+	ui() {
+		return {
+			input: 'input[type=number]',
+			label: 'label[for^="elementor-control-"]',
+		};
+	}
+
+	events() {
+		return {
+			'mousedown @ui.input': 'onMouseDownInput',
+			'mousedown @ui.label': 'onMouseDownLabel',
+			'mouseenter @ui.label': 'onMouseEnterLabel',
+			'mouseleave @ui.label': 'onMouseLeaveLabel',
+		};
+	}
+
+	scrub( input, movementEvent ) {
+		const movementType = this.getMovementType( movementEvent );
+
+		if ( SKIP_SCRUB === movementType ) {
+			return;
+		}
+
+		switch ( movementType ) {
+			case SCRUB_REGULAR:
+				input.value = +input.value + movementEvent.movementX;
+				break;
+
+			case SCRUB_ENHANCED:
+				input.value = +input.value + ( movementEvent.movementX * this.scrubSettings.enhancedNumber );
+				break;
+
+			default:
+				break;
+		}
+
+		// Fire an input event so other behaviors/validators can handle the new input
+		input.dispatchEvent( new Event( 'input', { bubbles: true } ) );
+	}
+
+	getMovementType( movementEvent ) {
+		if ( movementEvent.altKey ) {
+			this.skipperCount++;
+
+			// When ALT key is pressed, skipping x times before updating input value.
+			// 'skipperSteps' defines the exact times to skip, can be changed to any number.
+			if ( this.skipperCount <= this.scrubSettings.skipperSteps ) {
+				return SKIP_SCRUB;
+			}
+
+			this.skipperCount = 0;
+
+			return SCRUB_REGULAR;
+		}
+
+		return ( movementEvent.ctrlKey || movementEvent.metaKey ) ? SCRUB_ENHANCED : SCRUB_REGULAR;
+	}
+
+	isInputValidForScrubbing( input ) {
+		return input && ! input.disabled && 'number' === input.type;
+	}
+
+	onMouseDownInput( e ) {
+		const input = e.target;
+
+		if ( ! this.isInputValidForScrubbing( input ) ) {
+			return;
+		}
+
+		const trackMovement = ( movementEvent ) => {
+			this.scrub( input, movementEvent );
+		};
+
+		// For input, scrubbing effect works only after X time the mouse is down.
+		const checkIntentTimeout = setTimeout( () => {
+			clearTimeout( checkIntentTimeout );
+			document.addEventListener( 'mousemove', trackMovement );
+
+			$e.uiStates.set( 'document/scrubbing-mode', ScrubbingMode.ON );
+			input.classList.add( this.scrubSettings.scrubbingActiveClass );
+		}, this.scrubSettings.intentTime );
+
+		document.addEventListener( 'mouseup', () => {
+			document.removeEventListener( 'mousemove', trackMovement );
+			clearTimeout( checkIntentTimeout );
+
+			$e.uiStates.remove( 'document/scrubbing-mode' );
+			input.classList.remove( this.scrubSettings.scrubbingActiveClass );
+		}, { once: true } );
+	}
+
+	onMouseDownLabel( e ) {
+		const label = e.target;
+		const input = e.target.control;
+
+		if ( ! this.isInputValidForScrubbing( input ) ) {
+			return;
+		}
+
+		$e.uiStates.set( 'document/scrubbing-mode', ScrubbingMode.ON );
+		label.classList.add( this.scrubSettings.scrubbingActiveClass );
+		input.classList.add( this.scrubSettings.scrubbingActiveClass );
```

Comment:
Can you add it only to their parent?

Reply:
Done, as we decided - I created functions of add / remove classes, that receive an array of elements.